tests/integration_tests/run_integration_tests.py

```diff
@@ -45,9 +44,16 @@ def main():
 
     current_folder, current_file = os.path.split(__file__)
 
+    # Get test filenames from command line
+    parser = OptionParser()
+    (_, filenames) = parser.parse_args()
```

Comment:
Can we use argparse instead?
```suggestion
parser = argparse.ArgumentParser()
parser.add_argument("filenames",  action="append", nargs="*")
args = parser.parse_args()
```

Access `args.filenames`